tests/storage/datavolume.go

```diff
@@ -906,6 +912,9 @@ var _ = SIGDescribe("DataVolume Integration", func() {
 					Skip("Skip OCS tests when Ceph is not present")
 				}
 				dataVolume.Spec.PVC.StorageClassName = &sc
+			} else {
+				sc := tests.Config.StorageClassLocal
```

Comment:
Should this be tests.Config.RWOFileSystem instead? We are trying to get rid of StorageClassLocal in the long run.

Reply:
@awels specifically in this test it is devided to with OCS and without. if I use tests.Config.RWOFileSystem it will use the OCS.. I can hard code here 'local' insread of tests.Config.StorageClassLocal.. what do you think?

Reply:
The way I read the code above, the question is about does the storage support fstrim.  `[QUARANTINE][rfe_id:5070][crit:medium][vendor:cnv-qe@redhat.com][level:component]fstrim from the VM influences disk.img` However if the withOCS flag is true, we use a block device (which contradicts the description of the test btw, it should probably say how it influences the disk).

Anyway the 'withOCS' flag is basically just to indicate if we want to use block or not. So the flag should just toggle between RWOBlock storage class and RWOFileSystem storage class.

Reply:
I saw in line 862 in the function: getImageSize it uses a specific way to get the image size if it is OCS or not.. but  I saw maya removes the OCS check and use in a different PR. so I guess using RWOFileSystem will be good enough

Reply:
Yeah that `getImageSize` function actually runs some rbd commands to check for the actual size on the backend. That might be a problem for what we are trying to do, that is very specific to one kind of storage. The filesystem case should be fine with RWOFileSystem though.

Reply:
she removes it in this PR: https://github.com/kubevirt/kubevirt/pull/6942
so it wont be relevant anymore